osquery/tables/CMakeLists.txt

```diff
@@ -10,6 +10,9 @@ if(APPLE)
   )
 
   ADD_OSQUERY_LIBRARY(FALSE osquery_tables_darwin
+    applications/darwin/browser_chrome.cpp
+    applications/darwin/browser_firefox.cpp
```

Comment:
Both Chrome and Firefox are applicable to Linux distros, so perhaps we could generalize this seeing as only the `#define` paths need to change